xbmc/interfaces/builtins/LibraryBuiltins.cpp

```diff
@@ -62,6 +68,8 @@ static int CleanLibrary(const std::vector<std::string>& params)
     else
       CLog::Log(LOGERROR, "CleanLibrary is not possible while scanning for media info");
   }
+  else
+    CLog::Log(LOGERROR, "Unknown content type '{0}' passed to CleanLibrary, ignoring", params[0]);
```

Comment:
`params[0].c_str()`

Reply:
i was told to use libfmt and not pass by POD in an earlier review comment.

Reply:
not for `CLog::Log`, it doesn't use libmft and therefore doesn't compile (see jenkins log)

Reply:
alrightythen.

Reply:
still isn't fixed, you have done something wrong during updating the PR

Reply:
`{0}` also has to be replaced with `%s`

Reply:
a fix had been squashed wrongly (into the json commit) which reverted my fix once i moved the commit in history. fixed now.

Reply:
fixed now.